### DISCUSSION TITLE
Does psycopg2 2.9.x support Postgresql 17?

### DISCUSSION THREAD

**Discussion by krzysztofjeziorny** [psycopg/psycopg2#1733] [Q&A] [answered]
The [Features](https://www.psycopg.org/features/) page states that version 2

> Supports PostgreSQL from 7.4 to 17.

but I can't find it in the [release notes](https://www.psycopg.org/docs/news.html). Officially the support ends at 16.
Thanks for help!

**Comment by dvarrazzo** [comment, marked as answer]
I will put together release notes and will add postgres 17 to the test grid. However, working with psycopg 3, I know already that there will be no compatibility issue.
